src/web_parser/parser.py

```diff
@@ -0,0 +1,17 @@
+from papowerswitch_api import papowerswitch_api
+
+zip_code = 19473
```

Comment:
This should be something you can pass into the module of course; this is hard-coded to my zip code :) Overall looking much cleaner and modularized!

Reply:
Thanks! Agreed, I kept this file primarily to verify while I was refactoring. This file will likely be removed once things are looking squared away.